SYCL/ESIMD/api/functional/functions/functions_1d_select.hpp

```diff
@@ -0,0 +1,281 @@
+//===-- functions_1d_select.hpp - Functions for tests on simd rvalue select
+//      function. ---------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd 1d select function.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "../shared_element.hpp"
+#include "common.hpp"
+
+// for std::ceil
+#include <cmath>
+// for std::numeric_limits
+#include <limits>
+
+namespace esimd = sycl::ext::intel::experimental::esimd;
+
+namespace esimd_test::api::functional::functions {
+
+namespace details {
+
+constexpr int ceil(float arg) {
+  return (arg > static_cast<int>(arg)) ? arg + 1 : arg;
+}
+
+} // namespace details
+
+// Aliases to provide size or stride values to test.
+// This is the syntax sugar and they have the same type.
```

Comment:
As a suggestion:
```suggestion
// This is the syntax sugar, so no additional type safety here to expect
```
or 
```suggestion
// This is the syntax sugar just for code readability
```

Reply:
This comment has been applied [here](https://github.com/intel/llvm-test-suite/pull/875/commits/03d94c35b4a95e3ce26c99e1197bc50ccdc652aa#diff-295141c99d991cbb4971abc3bd754b2f17400cade3b579ac7bc36b28a1450dc3R36)